tasks/install-ui.yml

```diff
@@ -78,4 +78,4 @@
 
 - name: allow nginx to connect to consul (selinux)
   seboolean: name=httpd_can_network_connect state=yes persistent=yes
-  when: consul_is_ui and consul_install_nginx_config and ansible_selinux.status == "enabled"
+  when: consul_is_ui and consul_install_nginx_config and ansible_selinux == "true"
```

Comment:
Will this break again then with Ubuntu 16.04? (Possibly add a check for Ubuntu 14.04, which could then be merged upstream).